src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt

```diff
@@ -1,18 +1,18 @@
 package eu.kanade.tachiyomi.extension.en.anchira
 
 import eu.kanade.tachiyomi.source.model.SChapter
-import okhttp3.Response
 import java.util.Locale
 
 object AnchiraHelper {
     fun getPathFromUrl(url: String) = "${url.split("/").reversed()[1]}/${url.split("/").last()}"
 
-    fun prepareTags(tags: List<Tag>, group: Boolean) = tags.map {
+    fun prepareTags(tags: List<Tag>, group: Boolean) = tags.asSequence().map {
```

Comment:
Why convert to sequence when you need to sort them?

Reply:
Android Studio made me do it 🤐
![image](https://github.com/keiyoushi/extensions-source/assets/7104931/d9ff1ff0-2fd7-4479-89c8-157e721caefb)

Reply:
Just ignore it, it's not a compiler warning.

Reply:
Got it 👍